.github/workflows/transaction-sender-ecr-cd.yml

```diff
@@ -0,0 +1,28 @@
+name: Docker ECR Push - Transaction Sender
+
+on:
+  push:
+    branches: [main]
+    tags:
+      - v**
+  workflow_dispatch:
+
+
+permissions:
+  packages: write
+  contents: read
+  id-token: write
+
+# simplest example of using the rust-base action
+jobs:
+  docker-ecr-push:
+    uses: init4tech/actions/.github/workflows/ecr-build-and-push.yml@main
+    with:
+      rust-binary-name: transaction-submitter
+      environment: dev
+      requires-private-deps: true
+      dockerfile-path: transaction-submitter.Dockerfile
+    secrets:
+      aws-ecr-repository: ${{ secrets.AWS_ECR_REPOSITORY }}
+      aws-ecr-deployer-role-arn: ${{ secrets.AWS_ECR_DEPLOYER_ROLE_ARN }}
+      SSH_PRIVATE_KEY: ${{ secrets.SSH_PRIVATE_KEY }}
```

Comment:
delete this line if we have no more private deps, i have added the other two env secrets